addons/project/models/project.py

```diff
@@ -731,6 +743,8 @@ def action_view_tasks(self):
             .env.ref('project.act_project_project_2_project_task_all') \
             .sudo().read()[0]
         action['display_name'] = self.name
+        context = dict(safe_eval(action.get('context'), {'active_id': self.id}), create=self.active)
```

Comment:
And if the context is not defined in the action, we didn't have to define an empty dict as default value? 🤔 
 ```suggestion
         context = dict(safe_eval(action.get('context', {}), {'active_id': self.id}), create=self.active)
```